content/knowledge/scalinghistory.md

```diff
@@ -0,0 +1,144 @@
+If they are not on the same shard then they all needs to be ok with asynchronous updates which is [not ideal](https://www.coindesk.com/tech/2020/10/13/will-a-sharded-ethereum-be-flexible-enough-for-decentralized-finance/).
+
+## Avalanche
+
+The next level solution for scalability in terms of consensus algorithms is by Avalanche network which uses Snow Consensus with PoS as the underlying [sybil control mechanism](https://en.wikipedia.org/wiki/Sybil_attack).
+In Snow consensus, in short, a particular node randomly sub-samples from other nodes multiple times across multiple cycles and eventually arrives at a consensus.
+This is then recorded in a Directed Acyclic Graph (DAG) as opposed to a chain of blocks.
+Every node has its own view of the current consensus state; they do not progress in lock-step like they do for BFT algorithms.
+However, the local view of each node is eventually consistent with the view of all other nodes on the network.
+
+Snow consensus can easily scale to a higher level compared to PoW or PoS.
+Because the communicational complexity is much lower than BFT; we only need to communicate with random sub-samples of the network for each round.
+If these are logarithmically smaller than the entire set of validators on the network, the performance does not decrease exponentially with the size of the validator set.
+However, one common issue for PoS blockchains remains: the security of the network depends on the ratio of capital locked up for validation.
+Usually many PoS based networks impose a penalty for malicious validators or validators that aren’t online continuously by slashing which is taking away some of the staked tokens as a punishment.
+In case of Snow consensus, nodes monitor uptime and nodes do not receive rewards when they are not up long enough.
+
+Avalanche is a leaderless protocol, which is the first one of its kind.
+In BFT PoS, there is a leader for every round that has to propose the block; in Avalanche, everyone can propose state changes concurrently.
+
+## Flare
+
+Flare network combines best of both worlds by introducing FBA into the Snow consensus of Avalanche.
+Flare’s FBA achieves safety without relying too much on economic incentives that can interfere with high-value use cases.
+In Flare, each node has its own local set of validators or quorum set also known as Unique Node List (UNL).
+This varies from the usual FBA because of another requirement that there should be a minimum level of overlap between UNLs.
+This makes the network more single node failure resistant as well as Sybil attack resistant.
+This is because a malicious actor is unlikely to be able to spin up many node instances that will also make it to the UNL list of other existing quorum sets.
+The native token, Spark, can be used to delegate to the price providers for FTSO (link to FTSO article) without having any locking thereby enabling other usage of Spark at the same time.
```

Comment:
```suggestion
The native token, Spark, is used to delegate to the price providers for [FTSO](https://flare.xyz/ftso-a-breakdown/) without having any locking thereby enabling other usage of Spark at the same time.
```

Also, if this is the first time we mention the FTSO, we should not call it FTSO but `Flare Time Series Oracle (FTSO)` and only refer to it as FTSO from then on.